pom.xml

```diff
@@ -166,7 +171,7 @@
         <dependency> <!-- Use newer version than the one in core to speed the tests -->
             <groupId>org.jenkins-ci.modules</groupId>
             <artifactId>sshd</artifactId>
-            <version>2.0</version>
+            <version>2.3</version>
```

Comment:
Do we still need this dependency to be declared explicitly? It was there only because we ware getting 1.X otherwise which might not be needed after Jenkins version upgrade.

Reply:
Actualy no....